V1/Controllers/OrganisationController.cs

```diff
@@ -39,13 +40,25 @@ public async Task<IActionResult> GetOrganisation(string id)
             return StatusCode(resp.Meta.Code, resp);
         }
 
```

Comment:
Dont we need to keep this endpoint, otherwise the previous app versions will break?